include/tst_test.h

```diff
@@ -23,6 +23,9 @@
 #endif /* __TEST_H__ */
 
 #include <unistd.h>
+#ifndef __GLIBC__
+#include <limits.h>
+#endif
 
```

Comment:
The `limits.h` header is supposed to be included for `PATH_MAX` so there is no need to `#ifdef` it around.

But I would rather see the header included only in the testcases that actually needs the definition instead of the global `tst_test.h` header.